_posts/2020-02-27-react-conf-2020.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "ReactConf 2020"
+date:   2020-03-02
+tags: React Sydney
+author: Natalie Laing
+image: /img/blog/reactConf/react.png
+---
+
+## Introduction 
+React Conf AU is the first React conference in the southern hemisphere and it didn't disappoint. The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
+
+## Day 1
+![day 1](/img/blog/reactConf/conf_day1.jpg)
+
+### Keynote css in js
+Max Stoiber (Gatsby) 
+
+
+The father of styled components took us through CSS in js a history.
+Max expressed his passion for open source and encouraged everyone to create an open-source project and share them with him.
+
+60% of React installs now come with CSS in js.
+
+Max's pro tips
+```
+ Make it work, make it right, make it fast.
+
+ Share early, share often.
+
+ Document all the things. 
+```
+_Max Stoiber ReactConf 2020_
+
+### Themeability is the path to dark mode
+Diana Mounter (GitHub NYC)
+
+Diana walked us through how GitHub has been improving its themeability from using starting with two developers in 2017 to hiring their first react developer in 2018.
+Diana guided us on GitHub's journey from hardcoded hex color values to using variables and styled components.
+
+On the path to the ~~dark side~~ dark mode, Diana and her team realised that colour is relative and simply inverting the colours would not achieve true dark mode.
+
+``` colour is relative ``` _Diana Mounter ReactConf 2020_ 
+
+## (Proto)typing innovation in BBC News
+James Dooley (BBC news labs)
+
+How a team of artists, developers, and engineers get prototypes ready in 6 weeks for the BBC and how this team is always pushing the boundaries to keep the BBC relevant in today's tech-savvy world.
```

Comment:
get? Got?

Reply:
how they currently get their prototypes out. 
current process. 
happy to change.